db/migrate/20161011151936_create_course_excluded_exercises.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCourseExcludedExercises < ActiveRecord::Migration
+  def change
+    create_table :course_excluded_exercises do |t|
+      t.integer       :sequence_number,         null: false
+      t.uuid          :excluded_uuid,           null: false
+      t.uuid          :course_uuid,             null: false
```

Comment:
Totally cosmetic, but: I'd move this up to the top to make it easier to see what the sequence number is bound to.